dev-tools/ls-apis/src/system_apis.rs

```diff
@@ -51,15 +51,17 @@ pub struct SystemApis {
         BTreeMap<ServerComponentName, Vec<DepPath>>,
     >,
 
-    /// maps an API name to the server component that exposes that API
-    api_producers: BTreeMap<ClientPackageName, (ServerComponentName, DepPath)>,
+    /// maps an API name to the server component(s) that expose that API
+    api_producers: BTreeMap<ClientPackageName, ApiProducerMap>,
 
     /// source of developer-maintained API metadata
     api_metadata: AllApiMetadata,
     /// source of Cargo package metadata
     workspaces: Workspaces,
 }
 
+type ApiProducerMap = BTreeMap<ServerComponentName, DepPath>;
```

Comment:
I'm surprised this doesn't need to be a `Vec<DepPath>`.

Reply:
It does as of https://github.com/oxidecomputer/omicron/pull/7843/commits/dc84a1bf0ed849a9b2f066213caf86822531d2f4 :)